solidity/test/lib/mailboxes.ts

```diff
@@ -89,15 +96,16 @@ export async function dispatchMessageAndReturnMetadata(
 ): Promise<MessageAndMetadata> {
   // Checkpoint indices are 0 indexed, so we pull the count before
   // we dispatch the message.
```

Comment:
all this checkpoint stuff is kinda irrelevant to the mailbox now

Reply:
got it